client/src/components/Comments/index.tsx

```diff
@@ -48,35 +59,88 @@ class Comments extends React.Component<ICommentsProps> {
+                                        alt='avatar'
+                                        width={24}
+                                        height={24}
+                                        className='img-fluid rounded-circle mt-1 mr-1 mb-1'
+                                    />
+                                    <span className='mt-1'>{comment.authorId.username}</span>
+                                    {
+                                        comment.isEdit ?
+                                            (
+                                                <>
+                                                     <textarea
+                                                         rows={3}
+                                                         className='form-control'
+                                                         value={comment.newComment || comment.comment}
+                                                         onChange={
+                                                             (event: React.ChangeEvent<any>)
+                                                                 : void => this.props.changeComment(
+                                                                 event.target.value,
+                                                                 comment._id,
+                                                             )
+                                                         }
+                                                     />
+                                                    <div className='btn btn-danger mt-2'
+                                                         onClick={(): void => this.editComment(
+                                                             comment.newComment,
+                                                             comment._id,
+                                                             comment.authorId.email,
+                                                         )}
+                                                    >
+                                                        Change
+                                                    </div>
+                                                    <div className='btn btn-danger mt-2 ml-2'
+                                                         onClick={(): void => this.props.changeEditStatus(
+                                                             comment._id,
+                                                         )}
+                                                    >
+                                                        Cancel
+                                                    </div>
+                                                </>
+                                            )
+                                            :
+                                            this.props.user.email === comment.authorId.email &&
```

Comment:
You wrote "We check it on BE and on FE. Do we need it on FE ?"

then why we need it here?

Reply:
Fixed